src/Access/AccessControlHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Drupal\password_enhancements\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Access\AccessResultInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Session\SessionManagerInterface;
+
+/**
+ * Defines access checks for the password_enhancements module.
```

Comment:
Well this does not describe what this access checker does exactly :) It is in the password_enhancements module, so for sure, it does something for that ;P

Reply:
Yeah, well, I wanted to create this access handler as generic, that's why the generic description.

Its methods defines the actual functionality though.